tools/lint-docstring

```diff
@@ -7,6 +7,8 @@ import sys
 from pathlib import Path
 from typing import Dict, List, Tuple
 
+from zulipterminal.cli.run import in_color
```

Comment:
While color is nice, it doesn't always render well depending on the background, or in different situations like `tools/lint-docstring|less`. This would be cleanly solved if we used something like the `crayons` library.

Weren't the color changes previously in another commit? I think it'd be cleaner to have that separately, if we do end up doing so.

In terms of use in the main app, we could move to crayons there too, though ultimately I hope we can move to a loading screen instead, ie. load urwid very early, as explored in at least one PR.